ftl/core/notetypes.ftl

```diff
@@ -51,3 +51,5 @@ notetypes-error-generating-cloze = An error occurred when generating an image oc
 notetypes-error-getting-imagecloze = An error occurred while fetching an image occlusion note
 notetypes-error-loading-image-occlusion = Error loading image occlusion. Is your Anki version up to date?
 notetype-error-no-image-to-show = No image to show.
+notetype-create-edit-io-message = Tap the two squares to create or edit an occlusion.
+notetype-io-switch-type = Tap 'Type: Image Occlusion' to switch to normal text/media cards.
```

Comment:
I didn't think too carefully about the wording when adding this to the AnkiMobile beta. Maybe this would be better. 

```suggestion
notetype-io-switch-type = Change 'Image Occlusion' to 'Basic' or 'Cloze' to add normal text/media cards.
```

Reply:
TODO